### ISSUE TITLE
offline tutorial and documentation

### ISSUE THREAD

**Issue by skyvory** [fabricjs/fabric.js#2202]
Is there any offline tutorial and documentation to get?
It's quite an inconvenience to peek a glance on it while I'm not connected.

Edit: should've ask in google group according to guidelines
##

**Comment by jose920405** [comment]
While waiting for them to generate an official tutorial, for now you can call the fabric cdn inside a manifest.

Example:

your `html` file

```
<!DOCTYPE html>
<html manifest="canvas.manifest">
```

Your `canvas.manifest` file

```
CACHE MANIFEST
#comments

https://cdnjs.cloudflare.com/ajax/libs/fabric.js/1.7.3/fabric.min.js
```

**Comment by jose920405** [comment]
Any progress with this?

The manifest idea not works properly in release mode. We really need some idea of how to work in offline mode

**Comment by asturur** [comment]
what is the issue here?

**Comment by jose920405** [comment]
Provide documentation of how make this works in offline mode or in its absence, a reason why it is not possible

**Comment by asturur** [comment]
offline docs are not planned.
the docs are on the website and you can still cache it or build them
locally from fabricjs.com repository and spin up on localhost

**Comment by jose920405** [comment]
ok, thanks for reply

It's just what I've tried with the manifest. It has worked very well for other things, but when I downloaded the fabric cdn to a local path and loaded it, not only did it not work in offline mode, it does not work in release mode either.

that's why I thought that maybe there is some clarification about why these things happen in offline mode and because putting it in a local path does not work in release mode.
